game-settings.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>GameSelection</title>
```

Comment:
to już chyba było zmieniane w developie na Star Wars Quiz

Reply:
już poprawione